### DISCUSSION TITLE
v8 - Mise à jour des données OCS GE et GPU avec un ETL

### DISCUSSION THREAD

**Discussion by alexisig** [MTES-MCT/sparte#571] [Recettes]
### Données
* Ajout des données OCS GE de 17 départements  https://github.com/MTES-MCT/sparte/pull/554
  - Ain (01)
  - Aisne (02)
  - Alpes-de-Haute-Provence (04)
  - Alpes-Maritimes (06)
  - Charente-Maritime (17)
  - Côtes-d'Armor (22)
  - Dordogne (24)
  - Gard (30)
  - Hérault (34)
  - Lot-et-Garonne (47)
  - Lozère (48)
  - Oise (60)
  - Pyrénées-Atlantiques (64)
  - Paris (75)
  - Yvelines (78)
  - Somme (80)
  - Val-d'Oise (95)
* Mise à jour des zonages d'urbanisme et mise en place d'une mise à jour en continu https://github.com/MTES-MCT/sparte/pull/554
* Ajout d'un ETL pour ingérer, calculer et charger les données OCS GE https://github.com/MTES-MCT/sparte/pull/554

### Correctifs
* Correction d'un bug sur les surfaces des zonages d'urbanisme https://github.com/MTES-MCT/sparte/issues/560

**Full Changelog**: https://github.com/MTES-MCT/sparte/compare/v7.4.2...v8

<hr /><em>This discussion was created from the release <a href='https://github.com/MTES-MCT/sparte/releases/tag/v8'>v8</a>.</em>

**Comment by alexisig** [comment]
Les données de consommation des communes ayant fusionnées avec les communes suivantes ne sont pas disponibles : Pierre-Bénite (69152), Lugny-Bourbonnais (18131), Saint-Macoux (86231), Saint-Sulpice-de-Cognac (16355), Fleurigné (35112), Saint-Sigismond (49321), La Moncelle (08294), Saint-Sulpice-en-Pareds (85271), Cezais (85041), Les Trois Lacs (27058), Sombacour (25549), Goux-les-Usiers (25282), Gouzangrez (95282) et Urdès (64541)

On a crée un ticket pour régler ce problème : https://github.com/MTES-MCT/sparte/issues/569

**Comment by leabouche** [comment]
Est ce qu'on ne communiquerait pas sur l'ajout des 17 départements dans l'OCS GE dans la section "Nouveau" de la page d'accueil ?

**Reply by alexisig** [reply]
Tu verrais ça comment ? 

A mon avis garder l'info sur les rapports triennaux est important car il y a encore beaucoup de collectivités qui ne l'ont pas fait (même si la deadline était en août), mais on peut la déplacer ou mettre les deux infos au même endroit

**Reply by leabouche** [reply]
Ok! Dans ce cas faudrait voir si on a la place de mettre les deux infos l'une après l'autre (idéalement dans une autre section). Je mettrais qqch de très bref du genre "Les données OCSGE sont désormais disponibles dans 17 nouveaux départements"

**Reply by leabouche** [reply]
oui je suis d'accord, et surtout c'est pas clair qu'il y a deux nouveautés distinctes

**Reply by alexisig** [reply]
Un peu mieux mais je trouve toujours ça lourd dans la page

<img width="831" alt="image" src="https://github.com/user-attachments/assets/655a36ce-8e1f-4629-94c7-210767273d1a">
<img width="817" alt="image" src="https://github.com/user-attachments/assets/12d2bfdd-5d84-4e82-af8e-d330b5d802e4">

**Reply by leabouche** [reply]
En fait j'ai du mal à mesurer l'attente qu'il y a derriere ces données pour les users mais si on considère que c'est une nouveauté mineure on peut se contenter d'afficher les nouveaux départements d'une couleur différente sur la carte en bas de la page par ex - mais ca va passer plus inapercu

**Reply by smdsgn** [reply]
Pour moi ça doit faire l'objet d'une communication via lettre d'infos etc, je ne pense pas que ça ait beaucoup de sens de mettre un message sur les nouveaux départements disponibles d'autant plus qu'on va en ajouter toutes les semaines maintenant presque.

**Reply by Phil-Loriot** [reply]
ça sera le cas, lettre d'info, mais un message court avec lien vers la carte en haut de page d'accueil me semble bien aussi, d'autant que 17 d'un coup... c'est énorme !!!
après, on les aura 1 par 1
